### PR TITLE
[plg_search_tags]Code style

### PR DESCRIPTION
**Pull request by wojsmol**
### Summary of Changes

Code style changes for plg_search_tags
### Testing Instructions

code review

### CODE REVIEW

File: plugins/search/tags/tags.php [hunk 1]
```diff
@@ -102,15 +102,15 @@ public function onContentSearch($text, $phrase = '', $ordering = '', $areas = nu
...
-		$query->select('a.id, a.title, a.alias, a.note, a.published, a.access' .
-			', a.checked_out, a.checked_out_time, a.created_user_id' .
-			', a.path, a.parent_id, a.level, a.lft, a.rgt' .
-			', a.language, a.created_time AS created, a.description');
+		$query->select('a.id, a.title, a.alias, a.note, a.published, a.access'
+			. ', a.checked_out, a.checked_out_time, a.created_user_id'
+			. ', a.path, a.parent_id, a.level, a.lft, a.rgt'
+			. ', a.language, a.created_time AS created, a.description');
 
 		$case_when_item_alias = ' CASE WHEN ';
```
Comment (brianteeman) [inline plugins/search/tags/tags.php]: shouldnt this be indented one space as well?

Comment (wojsmol) [inline plugins/search/tags/tags.php]: IMHO not

Comment (brianteeman) [inline plugins/search/tags/tags.php]: Why not?You have made all the = below line up so why not this one

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on 1e8a4cf0d466c83ff4550a9df2b93c5841ea217b<br /><br />on code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10495">issues.joomla.org/joomla-cms/10495</a>.</sub>

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 1e8a4cf0d466c83ff4550a9df2b93c5841ea217b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10495">issues.joomla.org/joomla-cms/10495</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10495">issues.joomla.org/joomla-cms/10495</a>.</sub>

**Pull request merged**